packages/instrumentation-mongodb/package.json

```diff
@@ -16,12 +16,13 @@
     "compile:with-dependencies": "nx run-many -t compile -p @opentelemetry/instrumentation-mongodb",
     "compile": "tsc -p .",
     "prepublishOnly": "npm run compile",
-    "tdd": "npm run test-v5-v6-run -- --watch-extensions ts --watch",
-    "test": "npm run test-v5-v6 && npm run test-unit",
+    "tdd": "npm run test-v7 -- --watch-extensions ts --watch",
+    "test": "npm run test-v7 && npm run test-unit",
```

Comment:
Updating our default MongoDB version / and default tests to v7 means that the contributors will need to update their Node version, right? @david-luna, are there any concerns about it?